src/app/clients/view-client/view-client.component.ts

```diff
@@ -49,8 +50,10 @@ export class ViewClientComponent implements OnInit, OnDestroy {
   @ViewChild('sharesPaginator') sharesPaginator: MatPaginator;
   @ViewChild(MatSort) sort: MatSort;
 
-  constructor(private route: ActivatedRoute, private clientService: ClientsService,
-              private router: Router, public dialog: MatDialog) {}
+  constructor(private route: ActivatedRoute,
```

Comment:
You are not supposed to refactor the file in this PR. Please remove unrelated changes.